common/views/components/Collection/Collection.js

```diff
@@ -12,15 +13,39 @@ const Child = ({ child, currentWorkId }: childProps) => {
   return (
     <li
       key={child.path.path}
-      style={{
-        background:
-          child.work && child.work.id === currentWorkId ? '#298187' : '',
-        color: child.work && child.work.id === currentWorkId ? 'white' : '',
-        padding: child.work && child.work.id === currentWorkId ? '2px' : '',
-      }}
+      id={child.work ? `collection-${child.work.id}` : ''}
     >
-      {child.path.label || child.path.path}
-      {child.work && `: ${child.work.title}`}
+      {child.work && (
+        <NextLink href={`/works/${child.work.id}#collection-${child.work.id}`}>
+          <a
+            className="plain-link"
+            style={{
+              borderTop: '1px solid #298187',
+              padding: '5px',
+              display: 'block',
+              background:
+                child.work && child.work.id === currentWorkId ? '#298187' : '',
+              color:
+                child.work && child.work.id === currentWorkId ? 'white' : '',
+            }}
+          >
+            {child.path.label || child.path.path}
+            {child.work && `: ${child.work.title}`}
+          </a>
+        </NextLink>
+      )}
+      {!child.work && (
```

Comment:
For me it would be clearer if both these clauses were within the same `{...}` block (e.g. `{child.work ? ... : ...}`). Or is this a pattern used elsewhere on the client?